Can't merge -- you have got compilation failure (see Travis report on the PR page, which complains:)

```mvn
[ERROR] /home/travis/build/jcryptool/crypto/org.jcryptool.analysis.transpositionanalysis/src/org/jcryptool/analysis/transpositionanalysis/ui/wizards/TranspTextWizardPage.java:[374] 
[ERROR] 	lblTextshortened.setText(Messages.TranspTextWizardPage_0);
[ERROR] 	                                  ^^^^^^^^^^^^^^^^^^^^^^
[ERROR] TranspTextWizardPage_0 cannot be resolved or is not a field
```

maybe you forgot to push the newest version of that Messages.java?

--- a/org.jcryptool.analysis.transpositionanalysis/src/org/jcryptool/analysis/transpositionanalysis/ui/wizards/Messages.java
+++ b/org.jcryptool.analysis.transpositionanalysis/src/org/jcryptool/analysis/transpositionanalysis/ui/wizards/Messages.java
@@ -29,6 +29,7 @@ public class Messages extends NLS {
 	public static String AnalysisWizardIndexPage_simpleColumnAnalysisDescription;
 	public static String TranspTextModifyPage_textmodifydescription;
 	public static String TranspTextWizard_textwizard;
+	public static String TranspTextWizardPage_0;
 	public static String TranspTextWizardPage_columns;
 	public static String TranspTextWizardPage_pagedescription;
 	public static String TranspTextWizardPage_pagetitle;

